### ISSUE TITLE
visorinput: handle_locking_key(): remove extraneous parens

### ISSUE THREAD

**Issue by selltc** [davidker/unisys#49]
Source: Dan Carpenter dan.carpenter@oracle.com Mon 5/2/2016 5:53 AM

```
473                  int old_state = (test_bit(led, visorinput_dev->led));
```

> Remove extra parens.

See [KanBoard-1007](http://ustr-erl-6690.na.uis.unisys.com/?controller=task&action=show&task_id=1007&project_id=37)

**Comment by selltc** [comment]
Fixed by commit cc2f5a4b55ffeee565367a84c39df4abb939f63a.

**Comment by selltc** [comment]
Greg posted this to his staging-testing as [commit 73ba8afe33e22b4bae26098add458bed3e666bc2](https://git.kernel.org/cgit/linux/kernel/git/gregkh/staging.git/commit/drivers/staging?h=staging-testing&id=73ba8afe33e22b4bae26098add458bed3e666bc2).
